examples/src/examples/graphics/mesh-displacement.tsx

```diff
@@ -0,0 +1,225 @@
+import React from 'react';
+import * as pc from '../../../../';
+
+import { BindingTwoWay, LabelGroup, Panel, SliderInput, SelectInput, BooleanInput } from '@playcanvas/pcui/react';
+import { Observer } from '@playcanvas/observer';
+class MeshDisplacementExample {
+    static CATEGORY = 'Graphics';
+    static NAME = 'Mesh Displacement';
+    static WEBGPU_ENABLED = false;
```

Comment:
just curious - does it not work on WebGPU, or have you just not tried?

Reply:
ah I guess not due to the use of SHADOW_PCSS

Reply:
> just curious - does it not work on WebGPU, or have you just not tried?

Tried it without SHADOW_PCSS too but to many errors came up. So I thought it might be something else.

Reply:
yep fair enough, I'll check it later